src/ripplegw/waveforms/IMRPhenomD_NRTidalv2.py

```
@@ -33,7 +33,7 @@
 
 # The code below to compute the Planck taper is obtained from gwfast (https://github.com/CosmoStatGW/gwfast/blob/ccde00e644682639aa8c9cbae323e42718fd61ca/gwfast/waveforms.py#L1332)
 @jax.custom_jvp
-def get_planck_taper(x: Array, y: float) -> Array:
+def get_planck_taper(x: Float[Array, " n_freq"], y: float) -> Float[Array, " n_freq"]:
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Expected an expression

(F722)